lib/reporters/console.js

```diff
@@ -2,6 +2,7 @@
 
 var Tty = require('tty');
 var Diff = require('diff');
+var StringifySafe = require('json-stringify-safe');
```

Comment:
That's not what I want. Hoek only provides an error string instead, it's not because it's circular it cannot be diffed.